README.md

```diff
@@ -1,32 +1,43 @@
+- `filter_by_tenant`: list of specific tenants to which to dispatch. Can be a single tenant name, a list of comma separated tenant names or `\*`, meaning *all* (default). Since this parameter is a filter, the tenant names must be one of those defined within the `dispatches_file`.
+- `overwrite_version`: use this version instead of the one specified in the `dispatches_file`. Meaning, for each flavor to dispatch, instead of using its defined `version` parameter, this will be used instead. Can be a string of any value, so invalid versions are not checked for. This parameter accepts special keywords, see *Special version keywords* below.
+- `overwrite_env`: use this environment instead of the one specified in the `dispatches_file`. Meaning, for each flavor to dispatch, instead of using its defined `environment` parameter, this will be used instead. Can be a string of any value, so invalid environments are not checked for.
+- `overwrite_tenant`: use this tenant instead of the one specified in the `dispatches_file`. Meaning, for each flavor to dispatch, instead of using its defined `tenant` parameter, this will be used instead. Can be a string of any value, so invalid tenants are not checked for.
+- `default_releases_registry`: the registry to use for `release` type dispatches when no registry has been specified. Can be a string of any value, so invalid registries are not checked for.
+- `default_snapshots_registry`: the registry to use for `snapshots` type dispatches when no registry has been specified. Can be a string of any value, so invalid registries are not checked for.
+- `reviewers`: list of reviewers to add to the newly created state repo pull request. Can be a single Github username, a list of Github usernames or an empty string (default). Invalid or nonexistent usernames are not checked for.
+- `build_summary`: *contents* of the `build_summary.yaml` file outputted by the `build_images` workflow. If left blank (default), it will be automatically calculated by the workflow. See its documentation for more information.
+- `check_run_name`: name of the workflow check run where `build_summary` was uploaded. See the `build_images` documentation for more information.
 
-To create your own action, you can use this repository as a template! Just
-follow the below instructions:
 
-1. Click the **Use this template** button at the top of the repository
-1. Select **Create a new repository**
-1. Select an owner and name for your new repository
-1. Click **Create repository**
-1. Clone your new repository
+## Special version keywords
+
+`overwrite_version` and `dispatches_file.flavor.version` both accept a special set of keywords, which will be expanded during the make dispatches process. The accepted values are:
+
+- `$latest_release`: the latest avaliable release
+- `$latest_prerelease`: the latest avaliable prerelease
+- `$highest_semver_release_[semver]`: the release with the highest [semver] avaliable
```

Comment:
```suggestion
- `$highest_semver_release_[semver]`: the release with the **highest** [semver] digits available. See [semver precedence according Semver definition](https://semver.org/#spec-item-11): 1.0.0 < 2.0.0 < 2.1.0 < 2.1.1.  It could be used with `major` or `major.minor` digits, for example: `$highest_semver_release_v1`.
```